preferences/windows/2k12/requirements/requirements.yaml

```diff
@@ -4,7 +4,7 @@ kind: VirtualMachinePreference
 metadata:
   name: requirements
 spec:
-  # https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn303418(v=ws.11)
+  # https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn303418(v=ws.11)#processor
```

Comment:
Why is the url changed?

Reply:
Deep down, it is the same. The updated one just points you directly to the relevant information.